EIPS/eip-225.md

```diff
@@ -0,0 +1,474 @@
+---
+eip: 225
+title: Clique proof-of-authority consensus protocol
+author: Péter Szilágyi <peter@ethereum.org>
+discussions-to: https://github.com/ethereum/EIPs/issues/225
+status: Final
```

Comment:
Although this protocol has been up and running since Rinkeby. Let's start with merging this as draft Draft and move into Final through the Workflow described in [EIP-1](https://eips.ethereum.org/EIPS/eip-1) to perhaps get some final review comments.

Reply:
Done.